colabs/StatisticalTests.ipynb

```diff
@@ -0,0 +1,995 @@
+{
+ "nbformat": 4,
+ "nbformat_minor": 0,
+ "metadata": {
+  "colab": {
+   "provenance": [],
+   "collapsed_sections": [],
+   "toc_visible": true
+  },
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3"
```

Comment:
jeśli już to nie `list(runtime.toPandas()` tylko `runtime.toPandas().tolist()` ale i tak uważam, że powinniśmy zostać w obrębie szeregu pandasa

Reply:
zmieniłem na .tolist()